app/services/users.js

```diff
@@ -5,7 +5,15 @@ const ErrorMessages = require('../../config/error');
 
 exports.getUsers = async (offset, limit) => {
   try {
-    return await User.findAll({ offset, limit });
+    const user = await User.findAndCountAll({
+      limit,
+      offset
+    });
+    return {
+      page: parseInt(offset),
+      limit: parseInt(limit),
+      ...user
```

Comment:
Remember that when you return a list of an entity in this case of `user`, the list should contain the key that identifies it, in this case, `users`, currently, the list is being sent with the key rows.